src/Training.TruckWorld.Clone.Backend/TruckWorld.Infrastructure/Common/Mapper/NotificationMessageMapper.cs

```diff
@@ -0,0 +1,10 @@
+﻿using AutoMapper;
+
+namespace TruckWorld.Infrastructure.Common.Mapper;
+
+public class NotificationMessageMapper : Profile
```

Comment:
Why it's empty ?

Reply:
boshqa model orqali map qilinadi, u model hozircha kodda mavjud emas